apps/src/templates/HintsDisplay.jsx

```diff
@@ -31,6 +33,30 @@ module.exports = React.createClass({
     });
   },
 
+  renderBlocklyHint: function (hint) {
+    var node = document.getElementById(hint.hintId);
+    // Only render if the node exists in the DOM
+    if (node) {
+      Blockly.BlockSpace.createReadOnlyBlockSpace(node, hint.block);
+    }
```

Comment:
I see now that this is tricky because of how createReadOnlyBlockSpace and it being difficult to do this declaratively. Will try to noodle on this some more.